content/docs/user-guide/gems/reference/aws/aws-gamelift/cpp-api.md

```diff
@@ -115,13 +116,13 @@ void OnLeaveSessionAsyncComplete()
 
 ### Destroy Session (Passively)
 
-As the default behavior, when the last player leaves the game session, the game session will be destroyed.
+As the default behavior, when the last player leaves the game session, GameLift destroys the game session. 
```

Comment:
@onecent1101 Can we add a note about how the developer can change this?

Reply:
it is not in scope to support it with this release